backend/src/main/java/gov/cdc/usds/simplereport/api/Translators.java

```diff
@@ -284,6 +284,30 @@ public static Boolean parseYesNo(String v) {
   public static final Set<String> CANADIAN_STATE_CODES =
       Set.of("AB", "BC", "MB", "NB", "NL", "NT", "NS", "NU", "ON", "PE", "QC", "SK", "YT");
 
+  public static final Set<String> COUNTRY_CODES =
+      Set.of(
+          "AFG", "ALB", "DZA", "AND", "AGO", "ATG", "ARG", "ARM", "AUS", "AUT", "AZE", "BHS", "BHR",
```

Comment:
out of curiosity and future reference... do you have the resource of where you got this link handy? 🤔 Is this from Okta because I'm not seeing country here: https://reportstream.cdc.gov/resources/csv-schema 🧐 (and is this the best link to use to confirm the csv schema is correct?